packages/react/src/components/Modal/index.scss

```diff
@@ -8,6 +8,17 @@
   box-shadow: var(--c--components--modal--box-shadow);
   color: var(--c--components--modal--color);
   box-sizing: border-box;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  max-width: calc(100% - 2em - 6px);
+  max-height: calc(100% - 2em - 6px);
```

Comment:
Why this so precise calc ? Could it be broken according to design token ?

Reply:
I re-used the default CSS of a `dialog` element
<img width="367" alt="Capture d’écran 2024-04-23 à 11 18 50" src="https://github.com/openfun/cunningham/assets/9628870/832c381e-88e6-40fc-9b40-199b9b833c59">